src_erl/Communication_Layer/http_Nerlserver/src/source/sourceStatem.erl

```diff
@@ -205,9 +205,11 @@ roundRobin(ListOfSamples,CSVPath,Counter,[{ClientName,WorkerName,RouterHost,Rout
 sendSample([Head|ListOfSamples],CSVPath,Counter,ClientName,WorkerName,RouterHost,RouterPort)->
 %%  io:format("CSVPath ~p ,Counter ~p ,[{ClientName ~p ,WorkerName ~p ,RouterHost ~p ,RouterPort ~p ~n",[CSVPath,Counter,ClientName,WorkerName,RouterHost,RouterPort]),
```

Comment:
consider wrap with debug flag the printing 
better than using comment/uncomment